app/Helper.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App;
+
+class Helper
+{
+    public static function generateRandomString($length = 10)
```

Comment:
Why could you just used laravels helper instead? https://laravel.com/docs/8.x/helpers#method-str-random

Reply:
i didnt know that there's such a helper in laravel 😅 , thanks for letting me know